lib/x11regressiontest.pm

```diff
@@ -68,7 +68,8 @@ sub clean_shotwell {
 sub upload_libreoffice_specified_file {
 
     x11_start_program('xterm');
-    assert_script_run("wget " . autoinst_url . "/data/x11regressions/ooo-test-doc-types.tar.bz2 -O /home/$username/Documents/ooo-test-doc-types.tar.bz2");
+    type_string_slow("wget " . autoinst_url . "/data/x11regressions/ooo-test-doc-types.tar.bz2 -O /home/$username/Documents/ooo-test-doc-types.tar.bz2");
+    send_key "ret";
```

Comment:
So you don't want to test the result of the wget? What happen if it fails?
And why should not  do the 'return' inside `type_string_slow` by adding '\n' at the end? (but I agreed, it works ;-))

Reply:
What is the status for this? Is the wget can fail without any issue for the test?

Reply:
@ldevulder is right.

I consider this a bad idea. You are not checking the result of the wget and also type slower, "just to be safe"? If there is an openQA related problem why we need to type slower then we should be explicit about it not waste time unconditionally.

Reply:
Thanks @ldevulder  and @okurz 
I've added '\n' at the end of "type_string_slow" and removed "send_key "ret";" based on the first comment. Also added "assert_screen("libreoffice-wget-success");" to check if wget is successful. 
Since this PR has been merged. I will upload those changes in further PR.

Reply:
I don't think we should need to check the output of a script with a needle. This is what we have assert_script_run for. IMHO we should get rid of the type_string_slow here. It was added initially for the grub menue which has a limitation itself but within a full blown bash session there should be no problem to type with default speed unless we have a machine specific problem which should be solved with setting the test variable VNC_TYPING_LIMIT and not by typing slower in the test code